src/test/addrman_tests.cpp

```diff
@@ -50,6 +51,30 @@ class CAddrManTest : public CAddrMan
     }
 };
 
+static CNetAddr ResolveIP(const char* ip)
```

Comment:
Again, some handy helpers I can imaging being useful in more places than just the tests. Again, outside the scope of this PR.